### PR TITLE
Fix FAQ doc

### PR DESCRIPTION
**Pull request by chanseokoh**
`-Djib.console=plain` unnecessary with `--batch-mode` or `--console=plain`

### CODE REVIEW

File: jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessorTest.java [hunk 1]
```diff
@@ -58,6 +58,7 @@
 import org.junit.contrib.java.lang.system.RestoreSystemProperties;
 import org.junit.rules.TemporaryFolder;
 import org.junit.runner.RunWith;
+import org.mockito.Answers;
```
Comment (loosebazooka) [inline jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessorTest.java]: this file seems unrelated?

Comment (chanseokoh) [reply jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessorTest.java]: (blush) minor cleanup...
